### PR TITLE
NERT-218 My List View UI mostly completed, API/DB model updates

### PR DESCRIPTION
**Pull request by oscar-ip**
# Overview

## Links to Jira tickets

- NERT218,  NERT-227, NERT-226, NERT-224

## Description of relevant changes

- My list view mostly completed
- API changes to reflect data model changes for drafts to properly show in UI
- Fixed issue with access denied for non admin users

### CODE REVIEW

File: app/src/assets/images/planIcon.png
Comment (popkinj) [inline app/src/assets/images/planIcon.png:1]: May get a bit of overlap here.
The map is referencing these images from `/assets/icons/`

File: app/src/assets/images/projectIcon.png
Comment (popkinj) [inline app/src/assets/images/projectIcon.png:1]: Same as above

**Review by popkinj — APPROVED** [review]
Looks good. :+1:

File: app/src/assets/images/planIcon.png
Comment (oscar-ip) [reply app/src/assets/images/planIcon.png:1]: Yes, I wanted to talk to you about it :-)
React likes those assets within the /src folder, I will resolve this conversation and we can discuss further

**Pull request merged**